i18n/pt/campaigns/eoe/ice_and_death.po

```diff
@@ -26,3 +26,13 @@ msgid ""
 "\"All right, you all have your assignments. We will reach land within the hour. Be ready,\" Dr. Kensler says, dismissing the meeting. Before she can go below deck, you approach and remark that you aren't sure what you should be doing just yet.\n"
 "\"Oh, there will be enough manual labor to go around,\" she replies with the hint of a smirk."
 msgstr ""
+"Já se passaram longas oito semanas desde que você saiu de Boston, e deu muito trabalho. O Theodosia seguiu praticamente a mesma rota de seus predecessores, o Arkham e o Miskatonic - primeiro, navegando para o sul ao longo da costa leste e através do Canal do Panamá, antes de aventurar-se em direção ao Círculo Antártico. À medida que o tempo ficava cada vez mais frio, os icebergs se tornavam cada vez mais um problema para o navio navegar, mas com a experiência da tripulação e os registros dos capitães anteriores que fizeram a mesma viagem, você faz em um bom tempo. Por um breve momento, você vê os picos cobertos de névoa de Mts. Erebus e Terror à frente, marcando o local da Ilha de Ross, perto da plataforma de gelo que servirá como local de pouso da expedição.\n "
+"Conforme você se aproxima da costa, a Dra. Kensler e o resto da equipe se reúnem a bordo do convés do Theodosia para discutir seus planos. \"Usaremos o mesmo sistema de calça-bóia de antes para descarregar suprimentos na plataforma de gelo, \" ela explica. \"Sr. Fredericks, Sr. Ellsworth e Sr. Claypool, você irá levar um de nossos barcos menores para a plataforma para encontrar uma zona de desembarque. Sra. Takada, assim que tivermos um acampamento instalado e funcionando, esses aviões serão sua prioridade. Sr.Ashevak, espero que esses trenós ajudem a transportar os suprimentos do cais para o acampamento. Assim que chegarmos à plataforma de gelo, haverá muito trabalho a ser feito em pouco tempo. Não quero que você tenha que lutar com três dúzias de cachorros em cima de tudo isso. Portanto, coloque-os em ordem. \"\n"
```

Comment:
Yeah, so all new lines have to be replaced with \n, and then removed.  So this should be one really long line.

It's quite possible the github UI doesn't handle the long lines like this very well.  It might be better to download the file, and either edit with PoEdit or a free text editor like VSCode or Sublime